sktime/transformations/series/detrend/_deseasonalize.py

```diff
@@ -25,13 +23,39 @@
 class Deseasonalizer(_SeriesToSeriesTransformer):
     """A transformer that removes seasonal components from time series.
 
+    Fit computes seasonal components and stores them in `_seasonal`.
+
+    Transform aligns seasonal components stored in `_seasonal` with
+    the time index of the passed series and then
+    substracts them ("additive" model) from the passed series
+    or divides the passed series by them ("multiplicative" model).
+
     Parameters
     ----------
-    sp : int, optional (default=1)
+    sp : int, default=1
         Seasonal periodicity
-    model : str {"additive", "multiplicative"}, optional (default="additive")
+    model : str {"additive", "multiplicative"}, default="additive"
         Model to use for estimating seasonal component
 
+    Attributes
+    ----------
+    _y_index : Index
+        Time index of the series passed in fit.
+    seasonal_ : array of length sp
+        Seasonal components computed in seasonal decomposition.
+
+    See Also
+    --------
+    ConditionalDeseasonalizer
+
+    Notes
+    -----
+    - For further explanation on seasonal components and additive vs.
+    multiplicative models see
+    https://otexts.com/fpp3/components.html
+    - seasonal decomposition is computed using statsmodels
```

Comment:
Note section isn't rendering correctly. Try including the the links using the .rst syntax. E.g. `statsmodels <copy link here>`_

Reply:
Is there any way I can preview how the rendered version will look like?